app/views/mortgage_calculator/property_tax_calculator/show.html.erb

```diff
@@ -14,11 +14,15 @@
         <h1 class="stamp-duty__heading"><%= I18n.t("#{i18n_locale_namespace}.heading") %></h1>
         <h2 class="intro stamp-duty__subheading"><%= I18n.t("#{i18n_locale_namespace}.title") %></h2>
         <p>
-          <%= I18n.t(
-            "#{i18n_locale_namespace}.subtitle",
-            amount: second_home_threshold,
-            additional_property_rate: additional_property_rate
-          ) %>
+          <%= I18n.t("#{i18n_locale_namespace}.subtitle") %>
+          <a href="https://www.moneyadviceservice.org.uk/en/tools/house-buying/land-and-buildings-transaction-tax-calculator-scotland" target="_blank">
+            <%= I18n.t("#{i18n_locale_namespace}.subtitle_href_scotland") %>
+          </a>
+          <%= I18n.t("#{i18n_locale_namespace}.subtitle_href_separator") %>
+          <a href="https://www.moneyadviceservice.org.uk/en/tools/house-buying/land-transaction-tax-calculator-wales" target="_blank">
+            <%= I18n.t("#{i18n_locale_namespace}.subtitle_href_wales") %>
+          </a>
+          <%= I18n.t("#{i18n_locale_namespace}.subtitle_2") %>
```

Comment:
It would be better if we could use a single reference in the translation file for all this text rather than breaking it like this, which is possible by passing values for the country name and the link to the translation file from the view.

Reply:
I've refactored both translation files and not it only uses one element in the markup

Reply:
👍